src/pages/Detail/Detail.js

```diff
@@ -1,7 +1,113 @@
-import React from 'react';
-
+import React, { useState, useEffect } from 'react';
+import './Detail.scss';
+import DetailImage from './DetailComponent/DetailImages';
+import DetailSmallImage from './DetailComponent/DetailSmallImage';
+import TextInfo from './DetailComponent/TextInfo';
+import ColorImage from './DetailComponent/ColorImage';
+import SizeTag from './DetailComponent/SizeTag';
+import Button from './DetailComponent/Button';
 const Detail = () => {
-  return <div>fdsfds</div>;
+  const [detailSmallImage, setDetailSmallImage] = useState();
+  const [textData, setTextData] = useState();
+  const [colorData, setColorData] = useState();
+  const [sizeData, setSizeData] = useState();
+
+  useEffect(() => {
+    fetch('/data/detailSmallList.json')
+      .then(res => res.json())
+      .then(res => setDetailSmallImage(res))
+      .catch(e => console.error(e));
+  }, []);
+  useEffect(() => {
+    fetch('/data/textInfoData.json')
+      .then(res => res.json())
+      .then(res => setTextData(res))
+      .catch(e => console.error(e));
+  }, []);
+  useEffect(() => {
+    fetch('/data/detailColorList.json')
+      .then(res => res.json())
+      .then(res => setColorData(res))
+      .catch(e => console.error(e));
+  }, []);
+  useEffect(() => {
+    fetch('/data/sizeTag.json')
+      .then(res => res.json())
+      .then(res => setSizeData(res))
+      .catch(e => console.error(e));
+  }, []);
+
+  return (
+    <div className="container">
+      <span>남성복</span>
+      <div id="imageContainer">
+        <DetailImage />
+        <div className="smallImageList">
+          {detailSmallImage &&
+            detailSmallImage.map(Small => {
+              return (
+                <DetailSmallImage
+                  key={Small.id}
+                  id={Small.id}
+                  thumbnail={Small.thumbnail}
+                />
+              );
+            })}
+        </div>
+      </div>
+      <div className="detailInform">
+        <p className="title">무지막지·저지</p>
+        <div className="textContainer">
+          <div id="titleName">
+            <p>판매가</p>
+            <p>배송비</p>
+          </div>
+          <div className="textInfo">
+            {textData &&
+              textData.map(text => {
+                return (
+                  <TextInfo
+                    key={text.id}
+                    id={text.id}
+                    price={text.price}
+                    배송비={text.배송비}
+                  />
+                );
+              })}
```

Comment:
```suggestion
              textData.map(({id, price, 배송비}) => 
                  <TextInfo
                    key={id}
                    id={id}
                    price={price}
                    배송비={배송비}
                  />
                )}
```
- 매개변수자리에서 바로 구조분해할당 적용하면 좀 더 깔끔하겠네요
- arrow function에서 return을 바로하는 경우에는 중괄호 + return keyword 생략가능합니다!